packages/dx-vue-grid/src/components/table-layout/__snapshots__/virtual-table-layout.test.js.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`VirtualTableLayout should render correct layout 1`] = `
+<div
+  style="height: 100px;"
+>
+  <table
+    minwidth="400px"
+    style="min-width: 400px;"
+  >
+    <!---->
```

Comment:
Where is the table rows and columns?

Reply:
Render all hierarchical is very difficult

Reply:
@kvet done